src/chat/UnreadNotice.js

```diff
@@ -30,6 +30,11 @@ const styles = createStyleSheet({
     color: 'white',
     paddingRight: 4,
   },
+  unreadNumberExceeds: {
+      fontSize: 12,
```

Comment:
It looks like this new `styles.unreadNumberExceeds` is just meant to decrease the font size, from 14 to 12.

Can we find a solution that doesn't require an inconsistent font size? If the problem is that you're running into a maximum width of that `BRAND_COLOR`ed bubble, can we instead remove that constraint, and let the width grow just enough to show "99+" without shrinking the text?